nemo/utils/exp_manager.py

```diff
@@ -473,7 +473,7 @@ def get_log_dir(
                 logging.warning(
                     "No version folders would be created under the log folder as 'resume_if_exists' is enabled."
                 )
-                version = ""
+                version = None
```

Comment:
Ah this should have remained "". I'll fix this in another pr

Reply:
I was testing something. It works without error now. Let me try something before reverting it back. My suspicion is that w&b handler methods think we want to use version even when we pass "" as the version.

Reply:
Ok I'll leave that to you then, I haven't attempted the wandb fox yet.